content/en/docs/reference/configuration/multi-factor-authn/_index.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
We should say that the support is experimental and limited. In particular at the moment it's applicable only to login with username and password.